src/plot.js

```diff
@@ -175,7 +175,8 @@ export function plot(options = {}) {
   context.dispatchValue = (value) => {
     if (figure.value === value) return;
     figure.value = value;
-    figure.dispatchEvent(new Event("input", {bubbles: true}));
+    const Event = context.document.defaultView?.Event;
+    if (Event) figure.dispatchEvent(new Event("input", {bubbles: true}));
```

Comment:
Why make it optional? Feels like that would be surprising: Plot is not able to function without a complete implementation, and the user wouldn’t notice when it fails silently.

Alternative:

```suggestion
    figure.dispatchEvent(new context.document.defaultView.Event("input", {bubbles: true}));
```

Reply:
I was thinking of https://github.com/observablehq/plot/blob/09edfec1f26da02eeeb451f008dd2eb9fb8d8ffc/docs/components/PlotRender.js — but we wouldn't use this with tip: true

Reply:
well… we do
fixed in #2185